devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -319,7 +322,8 @@ class HumanStateProvider::impl::CmdParser : public yarp::os::PortReader
             if (command.get(0).asString() == "help") {
                 response.addVocab(yarp::os::Vocab::encode("many"));
                 response.addString("The following commands can be used to apply a secondary calibration assuming the subject is in the zero configuration of the model for the calibrated links. \n");
-                response.addString("Enter <calibrateAll> to apply a secondary calibration for all the links \n");
+                response.addString("Enter <calibrateAll> to apply a secondary calibration for all the links using the measured base pose \n");
+                response.addString("Enter <calibrateAllWithBase <baseLink>> to apply a secondary calibration for all the links assuming the <baseLink> to be in the origin \n");
```

Comment:
@lrapetti I am a bit unclear of the usage of this command. Does `calibrateAllWithBase` can be applied to any link to be considered as base link for consideration ? e.g. `calibrateAllWithBase RightHand`.

Or `calibrateAllWithBase` works with the link that is set as the base link. In this case, the input command `calibrateAllWithBase <baseLink>` is redundant, right ?

Reply:
This point should be more clear with the updated command discussed in https://github.com/robotology/human-dynamics-estimation/pull/237#issuecomment-786744471:
```
calibrateAllWithWorld <linkName>
```
it calibrate all the links, assuming the `<linkName>` is the world origin frame.